<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_selectFile">
       <property name="toolTip">
        <string>Select ASCII column-based file to load data</string>
       </property>
       <property name="statusTip">
        <string>Select ASCII column-based file to load data</string>
       </property>
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_selectFile">
       <property name="text">
        <string>Select File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_dateTime">
     <property name="title">
      <string>Datetime</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_columnIndex">
        <property name="toolTip">
         <string>Column to use as the row labels of the DataFrame (0-based)</string>
        </property>
        <property name="statusTip">
         <string>Column to use as the row labels of the DataFrame (0-based)</string>
        </property>
        <property name="text">
         <string>Column index</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_columnIndex">
        <property name="toolTip">
         <string>Column to use as the row labels of the DataFrame (0-based)</string>
        </property>
        <property name="statusTip">
         <string>Column to use as the row labels of the DataFrame (0-based)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_combineColumn">
        <property name="toolTip">
         <string>If [1, 2, 3] -&gt; try parsing columns 1, 2, 3 each as a separate date column. If [[1, 3]] -&gt; combine columns 1 and 3 and parse as a single date column. {‘foo’ : [1, 3]} -&gt; parse columns 1, 3 as date and call result ‘foo’ A fast-path exists for iso8601-formatted dates.</string>
        </property>
        <property name="statusTip">
         <string>If [1, 2, 3] -&gt; try parsing columns 1, 2, 3 each as a separate date column. If [[1, 3]] -&gt; combine columns 1 and 3 and parse as a single date column. {‘foo’ : [1, 3]} -&gt; parse columns 1, 3 as date and call result ‘foo’ A fast-path exists for iso8601-formatted dates.</string>
        </property>
        <property name="text">
         <string>Combine cols</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_combineColumn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>If [1, 2, 3] -&gt; try parsing columns 1, 2, 3 each as a separate date column. If [[1, 3]] -&gt; combine columns 1 and 3 and parse as a single date column. {‘foo’ : [1, 3]} -&gt; parse columns 1, 3 as date and call result ‘foo’ A fast-path exists for iso8601-formatted dates.</string>
        </property>
        <property name="statusTip">
         <string>If [1, 2, 3] -&gt; try parsing columns 1, 2, 3 each as a separate date column. If [[1, 3]] -&gt; combine columns 1 and 3 and parse as a single date column. {‘foo’ : [1, 3]} -&gt; parse columns 1, 3 as date and call result ‘foo’ A fast-path exists for iso8601-formatted dates.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>date_parser</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox_datetimeParser">
        <property name="toolTip">
         <string>Select format of the date-time column. This string will be forwarded to function &quot;datetime.strptime()&quot;</string>
        </property>
        <property name="statusTip">
         <string>Select format of the date-time column. This string will be forwarded to function &quot;datetime.strptime()&quot;</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Decimal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_decimal">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Character to recognize as decimal point. E.g. use ‘,’ for European data</string>
       </property>
       <property name="statusTip">
        <string>Character to recognize as decimal point. E.g. use ‘,’ for European data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Delimiter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_delimiter">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delimiter to use. Regular expressions are accepted</string>
       </property>
       <property name="statusTip">
        <string>Delimiter to use. Regular expressions are accepted</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Skip rows</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_skiprows">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Line numbers to skip (0-indexed) or number of lines to skip (int) at the start of the file</string>
       </property>
       <property name="statusTip">
        <string>Line numbers to skip (0-indexed) or number of lines to skip (int) at the start of the file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Use cols</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_useCols">
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Comma separated list of column indices (0-based) to be read. If this field is left emty - will read all columns</string>
       </property>
       <property name="statusTip">
        <string>Comma separated list of column indices (0-based) to be read. If this field is left emty - will read all columns</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_load">
     <property name="toolTip">
      <string>Actually load data</string>
     </property>
     <property name="statusTip">
      <string>Actually load data</string>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
